libfstrm: fix formality issues

James-TR opened the pull request:
Maintainer: me

Correct issues with TITLE and add URL pointing to the fstrm GitHub Page. Reformat the description so it renders correctly in menuconfig.

Fixes #9022

Signed-off-by: James Taylor <james@jtaylor.id.au>

neheb commented on libs/libfstrm/Makefile:
```diff
@@ -21,18 +21,19 @@ PKG_INSTALL:=1
-  SECTION:=libs
-  CATEGORY:=Libraries
-  TITLE:=This is fstrm, a C implementation of the Frame Streams data transport protocol.
+	SECTION:=libs
```
Normal indentation is 2 spaces.

James-TR replied:
I've removed those changes

James-TR commented:
Should the description have some spaces to indent it as well? I'm not sure on how it looks without any indentation.

BKPepe commented:
Ideally it should have. Remove dot in PKG_TITLE and it's good from my side.

James-TR commented:
@BKPepe thanks for the feedback! I've removed the dot and added two spaces indentation into the description after looking at other Makefiles for comparison.
